app/src/main/java/com/x8bit/bitwarden/data/vault/datasource/disk/VaultDiskSourceImpl.kt

```diff
@@ -125,7 +124,6 @@ class VaultDiskSourceImpl(
     override fun getDomains(userId: String): Flow<SyncResponseJson.Domains?> =
         domainsDao
             .getDomains(userId)
-            .filterNotNull()
```

Comment:
📓 Included as part of merge conflicts. Original change should have been in #4268 but was lost during rebase & conflict resolution.